web_monitoring/cli.py

```diff
@@ -53,7 +53,9 @@
 from tqdm import tqdm
 from urllib.parse import urlparse
 from web_monitoring import db
-from web_monitoring import internetarchive as ia
+import wayback
+# FIXME: importing a private API like this is not great :(
+from wayback._client import original_url_for_memento
```

Comment:
@danielballan any thoughts on how to best handle this?

- It might be nice to expose as a helper, but
- We could also solve the use-case here by adding the data `original_url_for_memento` gets onto the response objects returned by `get_memento`.

Reply:
Utility functions are inherently more reusable than utility methods. I like to consider the question, "Does this method need access to any internal state?" If no, lean toward function. Exposed publicly in https://github.com/edgi-govdata-archiving/wayback/pull/6

Reply:
That works!

FWIW, I was thinking of it more like attributes rather than exposing a method:

```py
memento = client.get_memento('http://web.archive.org/web/20170813195036/https://arpa-e.energy.gov/?q=engage/events-workshops')
assert memento.archived_url == 'https://arpa-e.energy.gov/?q=engage/events-workshops'
assert memento.date == datetime.datetime(2017, 8, 13, 19, 50, 36)
```

Reply:
(Obviously that still doesn’t need state, but makes a cleaner API since it’s common/relevant info a consumer might want, and definitely the sort of data we’d want to expose in edgi-govdata-archiving/wayback#2.)

Reply:
That's a solid point. Might be worth revisiting https://github.com/edgi-govdata-archiving/wayback/pull/6 again.